Tests/LibWeb/Layout/expected/table/border-spacing-rowspan.txt

```diff
@@ -133,34 +133,34 @@ ViewportPaintable (Viewport<#document>) [0,0 800x600]
       PaintableWithLines (BlockContainer(anonymous)) [8,8 784x0]
       PaintableWithLines (TableWrapper(anonymous)) [8,8 243.90625x257]
         PaintableBox (Box<TABLE>) [8,8 243.90625x257]
-          PaintableBox (Box<TBODY>) [9,9 161.90625x195] overflow: [9,9 221.90625x245]
-            PaintableBox (Box<TR>) [29,19 161.90625x39] overflow: [29,19 201.90625x88]
+          PaintableBox (Box<TBODY>) [29,19 201.90625x235]
+            PaintableBox (Box<TR>) [29,19 201.90625x39] overflow: [29,19 201.90625x88]
```

Comment:
Also here

Reply:
I opened up a bug today regarding this https://github.com/LadybirdBrowser/ladybird/issues/1790

Reply:
There's (proper) overflow here because of the rowspan="2" on the first cell:

![image](https://github.com/user-attachments/assets/db12e2b9-d3ae-430e-85ec-efb5a88c13e1)